### ISSUE TITLE
Husky Git hooks not installed on generated application

### ISSUE THREAD

**Issue by murdos** [jhipster/generator-jhipster#8457]
##### **Overview of the issue**

During the generation during the automatic npm install
>     husky > setting up git hooks
>     Can't find .git, skipping Git hooks installation.
>     Please check that you're in a cloned repository or run 'git init' to create an empty Git repository and reinstall husky.

##### **Motivation for or Use Case**

Git hooks are not installed, since husky is installed before git repository is initiated.

##### **Reproduce the error**

Generate a jhipster application with client (gateway or monolith).

##### **Suggest a Fix**

Run npm install after git init?

##### **JHipster Version(s)**

5.4.0

##### **JHipster configuration**

{
  "generator-jhipster": {
    "promptValues": {
      "packageName": "com.mycompany.myapp",
      "nativeLanguage": "en"
    },
    "jhipsterVersion": "5.4.1",
    "applicationType": "monolith",
    "baseName": "jhi_test_8457",
    "packageName": "com.mycompany.myapp",
    "packageFolder": "com/mycompany/myapp",
    "serverPort": "8080",
    "authenticationType": "jwt",
    "cacheProvider": "ehcache",
    "enableHibernateCache": true,
    "websocket": false,
    "databaseType": "sql",
    "devDatabaseType": "h2Disk",
    "prodDatabaseType": "mysql",
    "searchEngine": false,
    "messageBroker": false,
    "serviceDiscoveryType": false,
    "buildTool": "maven",
    "enableSwaggerCodegen": false,
    "jwtSecretKey": "NGRmMzgyZjVmOWJiOGFkOWJhZmM1OTQwOTg3YTRhNDE1NWIwMTA3NWI5OTUyMDIwZDExNzBlYmE5OTBjZDFkMTcxMTU4MGE0MDNiNDI5NTA2YjI0OTk5OTE1YTZiYTBhMDgxZDA4ZDdhOTE2MzRiMmE4MGJlMzk4MmEzZDkzMGU=",
    "clientFramework": "angularX",
    "useSass": true,
    "clientPackageManager": "npm",
    "testFrameworks": [],
    "jhiPrefix": "jhi",
    "enableTranslation": true,
    "nativeLanguage": "en",
    "languages": [
      "en"
    ]
  }
}

-   [x] Checking this box is mandatory (this is just to show you read everything)

**Comment by deepu105** [comment]
can you please update the issue template with more info so that someone can reproduce this? If you are confident its a bug and know how to fix it then do a PR

**Comment by jdubois** [comment]
Closing as PR is merged.

**Comment by deepu105** [comment]
I dont think that PR solves this issue(If its an issue after all, I have
not seen it)

Thanks & Regards,
Deepu

**Comment by murdos** [comment]
No, this issue is not solved. I created to distinguish it from #8448 solved by the linked PR.
I've added a jhipster configuration in the issue template to ease the reproduction (but I've just picked default options).

The other side effect of the described issue, is that prettier doesn't run on the generated files (no precommit hook, no automatic run of prettier).
Thus running manually prettier results in a lot of modified files, requiring a new commit.
